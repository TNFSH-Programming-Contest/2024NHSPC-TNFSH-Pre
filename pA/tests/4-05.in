731315889257951996 19451897796750309 150345326571910341
636984430176819202 59371340008201478 204757343441591966
