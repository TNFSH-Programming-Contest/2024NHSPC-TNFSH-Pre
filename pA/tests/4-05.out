192606594682834890
